src/main/java/games/strategy/triplea/delegate/BattleCalculator.java

```diff
@@ -1345,13 +1339,12 @@ public static int getUnitPowerForSorting(final Unit current, final boolean defen
       final Collection<TerritoryEffect> territoryEffects) {
     final boolean lhtrBombers = games.strategy.triplea.Properties.getLHTR_Heavy_Bombers(data);
     final UnitAttachment ua = UnitAttachment.get(current.getType());
-    int rolls;
+    final int rolls;
     if (defending) {
       rolls = ua.getDefenseRolls(current.getOwner());
     } else {
       rolls = ua.getAttackRolls(current.getOwner());
     }
```

Comment:
Those lines could be merged into `final int rolls = defending ? ui.getDefenseRolls(current.getOwner()) : ua.getAttackRolls(current.getOwner())`